### DISCUSSION TITLE
Lo spazio di Hassio

### DISCUSSION THREAD

**Discussion by Dregi56** [indomus/forum#277] [HUB: Home Assistant]
All'avvio di HA ho ricevuto una notifica di esaurimento dello spazio su HD: _Available space is less than 1GB!_  
Qualcuno mi può dire quanto spazio occupa Hassio? 
Se guardo in Supervisor-->Sistema-->Host mi trovo che sta occupando il **100%** del mio disco che è un SSD di ben **120 Gb**
L'attuale versione è **Versione core-2021.4.6** e mi viene proposto un aggiornamento che mi guardo bene dal fare in quanto probabilmente inchioderei tutto.
**Questi i componenti aggiuntivi:** DuckDNS, FTP, File editor, Home Ass. Google Drive BK, LogViewer, Mosquito, NodeRed, SSH&Terminal, Samba share, VisualStudioCode, Motion Eye.
Sono troppi per lo spazio a disposizione?
In questa ipotesi, ho provato a disinstallare VisualStudioCode e dato un riavvio dell'  host da Supervisor...
Disperazione, non parte più HA  non parte più... o almeno non riesco a collegarlo....
Ho cercato di "guardare" il disco collegandolo ad un PC con Ubuntu e trovo disk e disk 1, che, mi sembra hanno gli stessi contenuti.
Pingando dal prompt di windows l'Host su Raspberry, viene dato come _non raggiungibile_  però i pacchetti trasmessi e ricevuti corrispondono a 4 e i persi 0
Cosa c'è che non va?

**Comment by Giancky79** [comment]
https://www.home-assistant.io/more-info/free-space/

Controlla che non hai backup vecchi, db corrotto, addon non utilizzati, se usi web&ssh considera anche

*      docker image prune

Potresti liberare ulteriore spazio, pero fai sempre un backup prima

**Reply by Dregi56** [reply]
I backup? Non ne ho mai fatto uno se non quelli su Home Assistant Google Drive Backup che però non sono in locale!
Gli add-on sono quelli che ho citato. se ti riferisci alle integrazione ho quelle più comuni: Shelly, Life360, Raspberry, Tuya, vvado a memoria, qualcos'altro.... 
Se non è arabo certo è aramaico per me... cos'è _doker image prune_ ?

**Reply by Dregi56** [reply]
....si, con tutta la buona volontà, ora che non parte più come faccio a fare ciò che dici?
Mi sa che si va verso un _formattone_ purtroppo!!

**Comment by Giancky79** [comment]
Se non riparte (sei sicuro?) a volte staccare dalla presa e riattaccare potrebbe aiutare

**Reply by Dregi56** [reply]
Fatto. Nessun risultato. E' la prima volta che mi confronto con un malfunzionamento di hassio e devo dire che è frustrante per la poca possibilità di controllo della situazione....
Pensavo di collegare il Raspberry ad un monitor per vedere u n po' cosa scrive in fase di avvio e dove si inchioda...

**Comment by Nik71git** [comment]
Il database quanto grande è?

**Reply by Dregi56** [reply]
Scusa l'ignoranza, ma ho installato Hassio, dove potrei vedere la grandezza del database?.... a parte che ormai è tardi, non parte più, quindi...
Da Ubuntu su un diverso PC riesco a "leggere" l' HD, **oltre ai file autonation.yaml, configuration.yaml, script.yaml ecc. cosa altro (cartelle file ecc.)mi consigliate di mettere al sicuro per non perdere tutto,( ad esempio dove sono salvate le varie card di lovelance?) ... prima di formattare tutto e reinstallare il S.O.** ?
Da u n po' mi chiedevo.... da _Home Assistant Google Drive Backup_ come potrei recuperare un backup quando ad esempio ho una difficoltà come l'attuale?

**Reply by Newreader** [reply]
Sei sicuro di avere tutto lo spazio del disco a disposizione? Non hai per caso inizialmente copiato qualche immagine del sistema da una MicroSd magari di 8-16 GB? Potresti verificare con gparted come è strutturato il tuo disco. Se formatti e riproduci lo stesso ambiente non risolvi certamente e corri il concreto rischio che la situazione si possa ripetere. Io cercherei di indagare ulteriormente per cercare di capire la natura del problema.

**Reply by Nik71git** [reply]
il file si chiama home_assistant_v2.db e lo vedi tramite file editor. Puoi cancellarlo e riavviare, verrà ricreato.

**Comment by Dregi56** [comment]
Non so se faccio bene a postare ancora qui dentro...il titolo direbbe di sì.
Ho ripristinato HA da snapshot e tutto ok. C'è un problema che ancora una volta il disco si sta riempiendo in modo anomalo... ieri sera lo spazio occupato era di circa 30 Gb, oggi del doppio... qualcuno ha idea da cosa possa dipendere? Vorrei rimediare sta roba.

**Reply by matteocopelli** [reply]
A me era successo in passato utilizzando l'and-on di influxDB.
Secondo me in generale c'è qualche problema con un database che si riempie, passatemi il termine, all'infinito.
